### PR TITLE
Container resources

### PR DESCRIPTION
**Pull request by marcosdiez**
image

### CODE REVIEW

File: src/app/frontend/common/components/container/template.html [hunk 1]
```diff
@@ -247,7 +247,32 @@
       <kd-probe-card [probe]="container?.startupProbe"
                      [initialized]="initialized"> </kd-probe-card>
     </div>
+
+    <kd-property *ngIf="container?.resources?.limits"
+                 fxFlex="100">
+      <div key
+           fxFlex
+           class="kd-muted section-header"
+           i18n>Resources - Limits</div>
```
Comment (maciaszczykm) [inline src/app/frontend/common/components/container/template.html:256]: s/Resources - Limits/Resource Limits

File: src/app/frontend/common/components/container/template.html [hunk 2]
```diff
@@ -247,7 +247,32 @@
       <kd-probe-card [probe]="container?.startupProbe"
                      [initialized]="initialized"> </kd-probe-card>
     </div>
+
+    <kd-property *ngIf="container?.resources?.limits"
+                 fxFlex="100">
```
Comment (maciaszczykm) [inline src/app/frontend/common/components/container/template.html:252]: I would prefer to see both in one line, try to use 50% for both.

**Comment by marcosdiez** [comment]
image

**Comment by marcosdiez** [comment]
That's what it looks like. I like it.

**Comment by marcosdiez** [comment]
@maciaszczykm all done!

**Comment by marcosdiez** [comment]
@maciaszczykm when can we merge it ?

**Comment by floreks** [comment]
We have a soft code freeze due to https://github.com/kubernetes/dashboard/pull/7047 and to not cause major conflicts with the migration to the new architecture we do not merge new features currently.

**Comment by floreks** [comment]
Since https://github.com/kubernetes/dashboard/pull/7047 has been merged and it introduces major architecture changes we should close this PR. If you are willing to work on it, you can open a new PR against the current master.

/close

**Pull request closed without merging**